PolyEngine/RenderingDevice/OpenGL/Src/GLRenderingDevice.cpp

```diff
@@ -185,6 +185,40 @@ void GLRenderingDevice::EndFrame()
 {
 	glXSwapBuffers(this->display, this->window);
 }
+
+#elif defined(__APPLE__)
+
+// FIXME, why is this not working? For some reason Src-macOS folder is not on the include path for the project.
+#include "../Src-macOS/ObjCInterface.hpp"
```

Comment:
In CMakeLists.txt
`target_include_directories(polygldevice INTERFACE ${POLYGLDEVICE_INCLUDE}) `
needs to become
`target_include_directories(polygldevice PUBLIC ${POLYGLDEVICE_INCLUDE})`
(PUBLIC = PRIVATE + INTERFACE)
It's an oversight of mine that somehow worked until now.